util/unifycr/src/unifycr-env.c

```diff
@@ -52,6 +52,26 @@ int unifycr_read_env(unifycr_env_t *env)
     if (val)
         env->unifycr_mt = strdup(val);
 
+    val = getenv("UNIFYCR_META_DB_NAME");
+    if (val)
+        env->unifycr_meta_db_name = strdup(val);
+
+    val = getenv("UNIFYCR_META_DB_PATH");
+    if (val)
+        env->unifycr_meta_db_path = strdup(val);
+
+    val = getenv("UNIFYCR_SERVER_DEBUG_LOG");
+    if (val)
+        env->unifycr_server_debug_log = strdup(val);
+
+    val = getenv("UNIFYCR_META_SERVER_RATIO");
+    if (val)
+        env->unifycr_meta_server_ratio = strtoul(val, NULL, 0);
+
+    val = getenv("UNIFYCR_CHUNK_MEM");
+    if (val)
+        env->unifycr_chunk_mem = strtoul(val, NULL, 0);
+
```

Comment:
Shouldn't we be setting these in the configuration file now?

Reply:
I will create the configuration file entries, which can be overridden by the environmental variables.